examples/experimental/fs/fs.js

```diff
@@ -13,8 +13,24 @@ let file;
 })();
 
 export default async function () {
+	// About information about the file
 	const fileinfo = await file.stat();
 	if (fileinfo.name != "bonjour.txt") {
 		throw new Error("Unexpected file name");
 	}
+
+	// Define a buffer of the same size as the file
+	// to read the file content into.
+	const buffer = new Uint8Array(fileinfo.size);
+
+	// Read the file's content into the buffer
+	const bytesRead = await file.read(buffer);
+
+	// Check that we read the expected number of bytes
+	if (bytesRead != fileinfo.size) {
+		throw new Error("Unexpected number of bytes read");
+	}
+
+	// Seek back to the beginning of the file
+	await file.seek(0);
```

Comment:
I've tried to run this example and got a bunch of errors, like:
```
ERRO[0000] Uncaught (in promise) Error: Unexpected number of bytes read
        at file:///home/olegbespalov/projects/grafana/k6/k6/examples/experimental/fs/fs.js:31:8(33)  executor=shared-iterations scenario=default
ERRO[0000] Uncaught (in promise) Error: Unexpected number of bytes read
```

Which is expected since we have 100 VUs

But even running this with just one VU and one iteration gives us an error:

```
ERRO[0000] Uncaught (in promise) TypeError: seek() failed; reason: whence cannot be null or undefined  executor=shared-iterations scenario=default
```

Reply:
Turns out I forgot to pass the `whence` attribute 🤦🏻 
I've added it so the call looks like `await file.seek(0, SeekMode.Start)`, and it should pass on your side too now 👍🏻